root/etc/cont-init.d/98-ffmpeg

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/with-contenv bash
+
+echo "**** installing ffmpeg ****"
+apt update && apt install -y ffmpeg
```

Comment:
Is the apt update here needed? Does it make a difference?

Reply:
The lazylibrarian image ships without the package lists afaik, so without ffmpeg won't be found. I oriented myself on the other add-ons and only ran the update - if you want I can also do the proper "Docker-style" package install and clean up the package lists again afterwards.

Reply:
The way you have it is fine. Just wanted to check. You are correct the image is cleaned up. The mod should be fine like this though.

Reply:
@nemchik Yeah, whenever you want to install a package in a Ubuntu container you have to `apt update` first, not the case with Alpine and `apk` though.

Reply:
it should really be `apt-get update` and `apt-get install` so it doesn't complain about tty

also, @mKeRix please add the maintainer label to the dockerfile and list yourself like here: https://github.com/linuxserver/docker-mods/blob/code-server-nodejs/Dockerfile#L3

Reply:
@aptalca done!